scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -1,20 +1,24 @@
+version 1.0
+
 task PreprocessIntervals {
-    File? intervals
-    File? blacklist_intervals
-    File ref_fasta
-    File ref_fasta_fai
-    File ref_fasta_dict
-    Int? padding
-    Int? bin_length
-    File? gatk4_jar_override
-
-    # Runtime parameters
-    String gatk_docker
-    Int? mem_gb
-    Int? disk_space_gb
-    Boolean use_ssd = false
-    Int? cpu
-    Int? preemptible_attempts
+    input {
+      File? intervals
+      File? blacklist_intervals
+      File ref_fasta
+      File ref_fasta_fai
+      File ref_fasta_dict
```

Comment:
@ldgauthier noted we could remove this over in #6506.

Reply:
Oops, sorry, make that the closely named #6502.

Reply:
Sorry I just want to double-check what you meant. I just dug into the code path and saw that `--sequence-dictionary` is no longer required if `--reference` is supplied. However, the dictionary itself still needs to be there, ie if you have `ref.fasta`, then it expects `ref.fasta.dict` to exist as well. @ldgauthier can you clarify?

Reply:
Ah, that sounds right.

Reply:
Yes, it does still have to be in the input block so it gets localized since we don't implicitly localize any kind of indexes, but I see that you did take it out of the command line.  👍